src/core/bonded_interactions/angle_common.hpp

```diff
@@ -0,0 +1,104 @@
+/*
+  Copyright (C) 2010-2019 The ESPResSo project
+  Copyright (C) 2002-2010
+    Max-Planck-Institute for Polymer Research, Theory Group
+
+  This file is part of ESPResSo.
+
+  ESPResSo is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  ESPResSo is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef ANGLE_COMMON_H
+#define ANGLE_COMMON_H
+/** \file
+ *  Common code for functions calculating angle forces.
+ */
+
+#include "grid.hpp"
+#include "utils.hpp"
+
+/** Compute the normalized vector between two points.
+ *  @param[in]  r_mid     Position of the middle particle
+ *  @param[in]  r_out     Position of the outer particle
+ *  @param[out] vec       Normalized vector
```

Comment:
Probably it's a good idea to remove the output parameters and instead return a tuple.

Reply:
This function will be removed as per fweik's suggestion.